gallery/src/pageConfig.ts

```diff
@@ -240,6 +241,7 @@ const pageConfig: PageConfig = {
     'Collapsible Items': { content: NxCollapsibleItemsPage, type: 'react' },
     'Description List': { content: NxDescriptionListPage, type: 'react' },
     'List': { content: NxListPageV2, type: 'react' },
+    'Nivo': { content: NivoPage, type: 'react' },
```

Comment:
I suggest changing this to either "Nivo Charts" or just "Charts".  Just having it as "Nivo" is not helpful for people who don't already know what Nivo is.

Reply:
> I suggest changing this to either "Nivo Charts" or just "Charts". Just having it as "Nivo" is not helpful for people who don't already know what Nivo is.

Fixed here: 1df7d833c6515a9c9a8c7d151b0ff4bbe067661e